daemons/controld/controld_control.c

```diff
@@ -58,11 +58,11 @@ do_ha_control(long long action,
         crm_set_status_callback(&peer_update_callback);
         crm_set_autoreap(FALSE);
 
-        if (is_corosync_cluster()) {
 #if SUPPORT_COROSYNC
+        if (is_corosync_cluster()) {
             registered = crm_connect_corosync(cluster);
-#endif
         }
+#endif // SUPPORT_COROSYNC
```

Comment:
What I'd love to do eventually is replace calls like this with crm_cluster_t methods (which could vary by cluster layer). That would make all this moot (and much cleaner). So, I don't have a strong opinion one way or the other on changes like this one.

Reply:
That sounds like a great idea to me, for the future.